src/views/LottoView.js

```diff
@@ -0,0 +1,146 @@
+import RANK from '../constants/rank.js';
+
+const lottoView = {
+  bindBuyButtonEventHandler(onClickBuyButton) {
+    const buyButton = document.querySelector('#buy-button');
+    const buyMoneyInput = document.querySelector('#buy-money');
+
+    buyButton.addEventListener('click', event => {
+      event.preventDefault();
+
+      const buyMoney = Number(buyMoneyInput.value);
+
+      onClickBuyButton(buyMoney);
+      buyMoneyInput.value = null;
+    });
+  },
+
+  bindShowResultButtonEventHandler(onClickShowResultButton) {
+    const showResultButton = document.querySelector('#show-result-button');
+
+    showResultButton.addEventListener('click', event => {
+      event.preventDefault();
+
+      const bonusNumberInput = document.querySelector('#bonus-number');
+      const luckyNumbersInput = document.querySelectorAll('#lucky-numbers-input > input');
+      const bonusNumber = Number(bonusNumberInput.value);
+      const luckyNumbers = [...luckyNumbersInput].map(number => Number(number.value));
+
+      onClickShowResultButton(bonusNumber, luckyNumbers);
+    });
+  },
+
+  bindModalCloseButtonEventHandler(onClickModalCloseButton) {
+    const modalCloseButton = document.querySelector('#modal-close-button');
+
+    modalCloseButton.addEventListener('click', () => {
+      onClickModalCloseButton();
+    });
+  },
```

Comment:
```suggestion
    modalCloseButton.addEventListener('click', onClickModalCloseButton);
  },
```

context 때문에 arrow function 으로 감싸준거라면 어떻게 context 를 보존할 수 있을지 고민해보시면 좋겠습니다!

Reply:
컨트롤러에 있던 `bind~` 메서드들을 뷰로 일괄적으로 옮기면서  미처 확인하지 못했던 것 같습니다. arrow function으로 감싸주지 않아도 되는 거였네요! 수정하겠습니다~~